Source/WebCore/platform/graphics/WidthIterator.cpp

```diff
@@ -187,8 +187,9 @@ bool WidthIterator::hasExtraSpacing() const
 
 static void resetGlyphBuffer(GlyphBuffer& glyphBuffer, GlyphBufferStringOffset index)
 {
-    ASSERT(index < glyphBuffer.size());
-    glyphBuffer.shrink(index);
+    auto unsignedIndex = static_cast<unsigned>(index);
```

Comment:
Maybe we can assert that `index >= 0` before casting, just in case?

Reply:
Ok